lightning/src/ln/msgs.rs

```diff
@@ -1419,30 +1419,41 @@ mod fuzzy_internal_msgs {
 	// These types aren't intended to be pub, but are exposed for direct fuzzing (as we deserialize
 	// them from untrusted input):
 	#[derive(Clone)]
-	pub(crate) struct FinalOnionHopData {
-		pub(crate) payment_secret: PaymentSecret,
+	pub struct FinalOnionHopData {
+		pub payment_secret: PaymentSecret,
 		/// The total value, in msat, of the payment as received by the ultimate recipient.
 		/// Message serialization may panic if this value is more than 21 million Bitcoin.
-		pub(crate) total_msat: u64,
+		pub total_msat: u64,
 	}
 
-	pub(crate) enum OnionHopDataFormat {
-		NonFinalNode {
+	pub enum InboundPayload {
```

Comment:
`InboundPayload` and `OutboundPayload` seem somewhat generic names to have in the `msgs` module. Could we qualify these in some way or export them using a more meaningful module name?